### PR TITLE
feat: Rule of three command

### PR DESCRIPTION
**Pull request by NakajimaYoshi**
feat: Rule of three descent command

## Description

Uses the aviation rule of three to provide a rough estimate of TOD following a -3 degree path straight to the runway.
## Discord Username

HowNowBrownCrow#2591

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Update <small>_ May 2022</small>
 - chore:  add dependabot config (02/05/2022)
 
 Update <small>_ April 2022</small>
-
+- feat: add tod estimate calculator (30/4/2022)
```
Comment (NathanInnes) [inline .github/CHANGELOG.md:8]: Please adjust this to the appropriate month :)

File: src/commands/utils/rot.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        if (Number.isNaN(altitude)) {
+            await msg.channel.send({ embeds: [errorEmbed] });
+        } if (!msg.content.includes('FL') && altitude <= 1000) {
+            await msg.channel.send({ embeds: [flightLevelerrorEmbed] });
+        } if (altitude <= 1000 && text.startsWith('FL')) {
+            const todFL = Math.floor(altitude * (1 / 3) * (11 / 10));
+
+            const todFlightLevelEmbed = makeEmbed({
+
```
Comment (NathanInnes) [inline src/commands/utils/rot.ts:36]: Remove extra whitespace

File: src/commands/utils/rot.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    + 'See the official [guide](https://docs.flybywiresim.com/pilots-corner/beginner-guide/descent/#how-to-calculate-the-required-distance-for-descent) for planning an optimal descent.',
+                ]),
+            });
+
+            await msg.channel.send({ embeds: [todFlightLevelEmbed] });
+        } if (altitude >= 1000) {
+            const topOfDescent = Math.floor(altitude * (1 / 100) * (1 / 3) * (11 / 10));
+
+            const todEmbed = makeEmbed({
+
```
Comment (NathanInnes) [inline src/commands/utils/rot.ts:51]: Here also whitespace

**Comment by NakajimaYoshi** [comment]
Hello, are there any further changes requested? Thanks

**Comment by benw202** [comment]
> Hello, are there any further changes requested? Thanks

You still have some whitespace. Give me a ping on Discord if you need a hand :)

**Review by NathanInnes — APPROVED** [review]

**Comment by benw202** [comment]
image

Can we get this bug ironed out? I know it is a pretty unrealistic FL, but would be nice to have it displayed and calculated correctly!

**Comment by NakajimaYoshi** [comment]
Closed due to utility redundancy

**Pull request closed without merging**